code401challenges/src/main/java/tree/Tree.java

```diff
@@ -69,4 +69,22 @@ protected void postOrderHelper(Node currentNode, ArrayList values){
             values.add(currentNode.getValue());
         }
     }
+
+//  Finds the max value in Tree
+    public ArrayList<T> findMax(){
+        ArrayList<T> output = new ArrayList<>();
+        if(this.root == null){
+            return output;
+        }
+        findMaxValueHelper(this.root, output);
+        return output;
+    }
+
+    protected void findMaxValueHelper(Node<T> currentNode, ArrayList output){
+       if(currentNode != null){
+           findMaxValueHelper(currentNode.leftChild, output);
+           findMaxValueHelper(currentNode.rightChild, output);
+           output.add(currentNode.getLeftChild());
```

Comment:
Since this should work on all trees, you don't want to just look at the left child. You'll instead need to do some type of comparison between the `currentNode`'s value and the highest values you've found up to that point.